src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md

```diff
@@ -1,18 +1,30 @@
-### Example 1: {{ Add title here }}
+### Example 1
 ```powershell
-PS C:\> {{ Add code here }}
-
-{{ Add output here }}
+Import-Module Microsoft.Graph.Applications
+$params = @{
+	KeyCredential = @{
+		Type = "AsymmetricX509Cert"
+		Usage = "Verify"
+		Key = [System.Text.Encoding]::ASCII.GetBytes("MIIDYDCCAki...")
+	}
+	PasswordCredential = $null
+	Proof = "eyJ0eXAiOiJ..."
+}
+Add-MgApplicationKey -ApplicationId $applicationId -BodyParameter $params
 ```
-
-{{ Add description here }}
```

Comment:
Missing description:
```
{{ TODO: Add description here }}

```

Reply:
Are you failing because the descriptions are missing?

Reply:
And as a rule they mostly have Example 1 Example 2 etc as the title

Reply:
The validation rule is by Azure's AutoREST. It expects all examples to have a tile and description. See https://github.com/Azure/autorest.powershell/blob/1a1c6d89c2b07be22fcada5cd3a13e92b1a7e1c7/powershell/resources/assets/tools/Resources/how-to.md#generating-documentation